docs/user-guide/commands/argocd_gpg_add.md

```diff
@@ -8,6 +8,14 @@ Adds a GPG public key to the server's keyring
 argocd gpg add [flags]
 ```
 
+### Examples
+
+```
+
+		# Add a GPG public key to the server's key
+		argocd gpg add --key <GPG_PUBLIC_KEY>
```

Comment:
Could you either un-indent the string or use the `templates.Examples` function to correct indentation?

Reply:
YEs I have done the changes for all `.md` files

Reply:
If I run `make clidocsgen` command then it is adding the un-indentation to the string
So is it necessary to run `make clidocsgen` in this case?

Reply:
You have to remove the indentation in the .go files and then run `make clidocsgen`.

So, 


```go
		Example: `
# Remove a GPG public key from the server's key using its key ID 
argocd gpg rm <KEYID>
		`,
```

Looks weird, but it gets the job done :-)

Reply:
Got it! Thanks for the suggestions 
I hope all checks will pass now